app/(auth)/register/page.tsx

```diff
@@ -1,7 +1,183 @@
 "use client";
 
+// @Nextjs
+import Image from "next/image";
+import Link from "next/link";
+
+// @Shadcn
+import { Button } from "@/components/ui/button";
+import { Checkbox } from "@/components/ui/checkbox";
+
+import {
+  Form,
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+
+// @View Models
+import LoginViewModel from "./RegisterViewModel";
```

Comment:
change to `import RegisterViewModel from "./RegisterViewModel";` because is the Register page